src/hooks/screen/useCreateChat.ts

```diff
@@ -118,10 +114,10 @@ const useCreateChat = () => {
       const chatData = createChannelJson(initChannel, selectedUser);
 
       const channelList = ChannelList.fromMessageSignedAPI(chatData);
-      channelList.saveIfLatest(localDb);
-      handleMemberSchema(initChannel);
+
+      await Promise.all([channelList.saveIfLatest(localDb), handleMemberSchema(initChannel)]);
       setLoadingCreateChat(false);
-      goToChatScreen(channelList, from);
+      goToChatScreen(channelList as ChannelListObject, from);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk are good. Wrapping the two async operations `channelList.saveIfLatest(localDb)` and `handleMemberSchema(initChannel)` with `Promise.all()` ensures that both operations complete before proceeding, which could potentially improve performance by allowing these operations to run concurrently.



<!-- This is an auto-generated comment by OSS CodeRabbit -->